Dockerfile

```diff
@@ -6,14 +6,14 @@
 # docker_parent_image is the base layer of full and jre image
 #
 # Use latest version here: https://github.com/orgs/openzipkin/packages/container/package/alpine
-ARG docker_parent_image=ghcr.io/openzipkin/alpine:3.14.3
+ARG docker_parent_image=ghcr.io/openzipkin/alpine:3.15.3
 
 # java_version is hard-coded here to allow the following to work:
 #  * `docker build https://github.com/openzipkin/docker-java.git`
 #
 # When updating, also update the README
 #  * Use current version from https://pkgs.alpinelinux.org/packages?name=openjdk15
-ARG java_version=15.0.5_p3
+ARG java_version=15.0.6_p5-r0
```

Comment:
I don't think we want `-r0` here. From what I recall it should just be `15.0.6_p5`.